src/index.ts

```diff
@@ -431,7 +431,7 @@ export class ValidationService {
             // Allows developers to override the default MVC Providers by adding custom providers BEFORE bootstrap() is called!
             return;
         }
-        this.logger.log(`Registered provider: ${name}`);
+        this.logger.log(`Registered provider: %s`, name);
```

Comment:
nit: you don't need interpolation now

```suggestion
        this.logger.log("Registered provider: %s", name);
```